exercises/any_odds_and_evens/main_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import "testing"
+
+func TestAnyIsOdd(t *testing.T) {
+	for _, tc := range []struct {
+		s    []int
+		want bool
+	}{
+		{[]int{}, false},
```

Comment:
Please add tests with zeros and a test with with a slice containing a single negative integer.